src/configmanager.cpp

```diff
@@ -295,9 +294,6 @@ bool ConfigManager::load()
 bool ConfigManager::reload()
 {
 	bool result = load();
-	if (transformToSHA1(getString(ConfigManager::MOTD)) != g_game.getMotdHash()) {
-		g_game.incrementMotdNum();
-	}
 	return result;
```

Comment:
😄 
```suggestion
	return load();

Reply:
This function can be removed altogether. Just `load()` again.